### ISSUE TITLE
Getting these warnings with the new typescript version

### ISSUE THREAD

**Issue by camiloazula** [leebyron/iterall#11]
```
    ERROR in [default] /Volumes/ssd-sensitive/projects/twinatom/twinatom-engine-seed/node_modules/iterall/index.d.ts:22:49 
    Cannot find name 'TValue'.
    
    ERROR in [default] /Volumes/ssd-sensitive/projects/twinatom/twinatom-engine-seed/node_modules/iterall/index.d.ts:22:68 
    Cannot find name 'TValue'.
    
    ERROR in [default] /Volumes/ssd-sensitive/projects/twinatom/twinatom-engine-seed/node_modules/iterall/index.d.ts:26:60 
    Type expected.
    
    ERROR in [default] /Volumes/ssd-sensitive/projects/twinatom/twinatom-engine-seed/node_modules/iterall/index.d.ts:26:62 
    '{' or ';' expected.
    
    ERROR in [default] /Volumes/ssd-sensitive/projects/twinatom/twinatom-engine-seed/node_modules/iterall/index.d.ts:26:65 
    'Iterator' only refers to a type, but is being used as a value here.
    
    ERROR in [default] /Volumes/ssd-sensitive/projects/twinatom/twinatom-engine-seed/node_modules/iterall/index.d.ts:26:74 
    Cannot find name 'any'.
    
    ERROR in [default] /Volumes/ssd-sensitive/projects/twinatom/twinatom-engine-seed/node_modules/iterall/index.d.ts:28:2 
    Expression expected.
```

**Comment by camiloazula** [comment]
#12 fixes this one

**Comment by leebyron** [comment]
released in 1.0.3
